apps/modernization-ui/src/apps/page-builder/components/CreateQuestion/CreateQuestion.tsx

```diff
@@ -142,61 +144,90 @@ export const CreateQuestion = ({ modalRef, question }: any) => {
         fetchCodeSystemOptions();
     }, []);
 
-    const buildOptions = (optionsToBuild: any[]) =>
-        optionsToBuild.map((opt: string) => (
-            <option value={opt} key={opt}>
-                {opt}
-            </option>
-        ));
-    const buildCodeOptions = (optionsToBuild: any[]) =>
-        optionsToBuild.map((opt: any) => (
-            <option value={opt.value} key={opt.value}>
-                {opt.label}
-            </option>
-        ));
-    const handleSubmit = () => {
-        if (question?.id) return handleUpdateQuestion();
+    const onSubmit = handleSubmit(async (data) => {
+        if (question?.id) return handleUpdateQuestion(data);
         const request = {
             dataMartInfo: {
                 defaultLabelInReport: 'Test',
```

Comment:
Issue: Hard coded value for `defaultLabelInReport`, however this does not correctly map to the API. See above comment